content/about/about-jobs.md

```diff
@@ -8,7 +8,7 @@ aliases = ["/jobs/"]
 +++
 ### Current openings:
 
-* [Membership Manager](#membership-manager)
+* Membership Manager](#membership-manager)
```

Comment:
This looks like a typo. Is the branch up to date?

Reply:
That is a typo. I second guessed myself after merging with master. Fixed with the latest commit: https://github.com/spotlightpa/poor-richard/pull/368/commits/67898a9a3c0df133604829146f125bf994de406f